util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -165,6 +167,24 @@
   public static final HadoopConfigurationProperty<String> IMPERSONATION_SERVICE_ACCOUNT_SUFFIX =
       new HadoopConfigurationProperty<>(".auth.impersonation.service.account");
 
+  /**
+   * Key prefix for the user identifier associated with the service account to impersonate when
+   * accessing GCS.
+   */
+  public static final HadoopConfigurationProperty<Map<String, String>>
+      USER_IMPERSONATION_IDENTIFIER_PREFIX =
```

Comment:
Rename to `IMPERSONATION_SERVICE_ACCOUNT_FOR_USER_SUFFIX` for consistency with IMPERSONATION_SERVICE_ACCOUNT_SUFFIX`?